pkg/config/process.go

```diff
@@ -35,8 +48,9 @@ func setupProcesses(config Config) {
 	config.SetKnown("process_config.dd_agent_env")
 	config.SetKnown("process_config.enabled")
 	config.SetKnown("process_config.intervals.process_realtime")
-	config.SetKnown("process_config.queue_size")
-	config.SetKnown("process_config.rt_queue_size")
+	procBindEnvAndSetDefault(config, "process_config.queue_size", DefaultCheckQueueSize)
+	procBindEnvAndSetDefault(config, "process_config.process_queue_bytes", DefaultProcessQueueBytes)
+	procBindEnvAndSetDefault(config, "process_config.rt_queue_size", DefaultRTCheckQueueSize)
```

Comment:
Do we really want to add `DD_PROCESS_AGENT`-prefixed environment variables? I don't think we need to add an (legacy?) alias for settings that never had an associated environment variable before.

Reply:
Thanks for pointing this out @vickenty. So originally `process-agent` env vars were prefixed with `DD_PROCESS_AGENT_`. In the [process-agent config refactor RFC](https://github.com/DataDog/architecture/pull/953#discussion_r759672755) we discussed about being more consistent with the `datadog.yaml` file and use `DD_PROCESS_CONFIG_`.

In order to avoid any confusion for clients expecting the old pattern to work, we decided to support both `DD_PROCESS_AGENT_` and `DD_PROCESS_CONFIG_` prefixes.